res/.htaccess

```diff
@@ -1,50 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Initially I didn't remove it because I was considering still using my local apache server. But with deploy previews this is much less useful, and we also have a `yarn start-prod` tool that can help. so :+1: